src/main/java/org/jboss/forge/arquillian/command/TestSetUp.java

```diff
@@ -0,0 +1,15 @@
+package org.jboss.forge.arquillian.command;
+
+
+import org.jboss.forge.roaster.model.source.JavaClassSource;
+import org.jboss.forge.roaster.model.source.MethodSource;
+
+public interface TestSetUp {
+
+    default void removeTestMethod(String methodName, JavaClassSource test) {
```

Comment:
In addition, as we pass method as string this might pose a risk in the future that when we rename it in the template we will loose a chance to spot this gap in our implementation. I would propose to write a test for it too.